faucet.js

```diff
@@ -8,6 +8,21 @@ const faucetContract = require("./build/contracts/SponsorFaucet.json");
 const gas_estimation_ratio_withdraw = 1.8;
 const gas_estimation_ratio_default = 1.3;
 
+/**
+ * @dev parse encoded data
+ * @param data
+ */
+function _parse(data) {
```

Comment:
What to parse? Strongly recommend to use a meaningful function name.

Reply:
modified